wcfsetup/install/files/js/wcf.globalHelper.js

```diff
@@ -289,6 +289,24 @@
 		return obj.hasOwnProperty(property);
 	};
 	
+	window.deprecatedFeature = function(message) {
+		console.log("A deprecated feature is used, while 'ENABLE_DEPRECATION_WARNINGS' is enabled: " + message);
```

Comment:
This creates a significant nuisance during normal operations, these should be suppressed unless `ENABLE_PRODUCTION_DEBUG_MODE` is enabled.

Also, we should consider including `new Error().stack` as an additional parameter, otherwise this message will be a nightmare to debug.